src/main/java/cat/tophat/creepycreepers/common/entities/CreepyCreeperEntity.java

```diff
@@ -27,18 +27,18 @@
 	 * 
 	 * @param type The entity type.
 	 * @param worldIn The current world.
-	 * @param ingitedSound A supplier of the ignition sound.
+	 * @param ignitedSound A supplier of the ignition sound.
 	 */
-	public CreepyCreeperEntity(EntityType<? extends CreepyCreeperEntity> type, World worldIn, Supplier<SoundEvent> ingitedSound) {
+	public CreepyCreeperEntity(EntityType<? extends CreepyCreeperEntity> type, World worldIn, Supplier<SoundEvent> ignitedSound) {
 		super(type, worldIn);
-		this.ignitedSound = Lazy.of(ingitedSound);
+		this.ignitedSound = Lazy.of(ignitedSound);
 	}
 
 	/**
 	 * Plays the step sound when the mob walks over a certain block.
 	 * 
 	 * @param pos   The position to play the sound at.
-	 * @param block The block under the mob.
+	 * @param blockIn The block under the mob.

```

Comment:
The only point of putting In on a parameter is if you have two parameters with the same name, and even then you should have more specific parameter names.

Reply:
True - I was just going off of the name already there to solve the build error.